core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -54,6 +56,18 @@ var PreferencesPage = function() {
     options.first().click();
   };
 
+  this.selectPreferredAudioLanguage = function(language) {
+    preferredAudioLanguageSelector.click();
+    var optionsList = element.all(by.css('.select2-results'));
```

Comment:
Probably this should go at the top of the file (near lines 23-33)

Reply:
Done!